File: DPIconFontExample/ViewController.m
//
//  ViewController.m
//  DPIconFontExample
//
//  Created by 张鹏 on 2021/1/14.
//

#import "ViewController.h"
#import <DPIconFont/DPIconFont.h>

@interface ViewController ()

@property (weak, nonatomic) IBOutlet UILabel *textLabel;

@property (weak, nonatomic) IBOutlet UIImageView *imageView;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.textLabel.font = ICFFont(32);
    self.textLabel.text = @"\U0000e602";
    self.textLabel.text = ICFSymbol(@"&#xe602");
    
    self.imageView.image = ICFSymbolImage(@"\U0000e602", 30);
    self.imageView.tintColor = UIColor.redColor;
}

@end

File: DPIconFontExample-Swift/ViewController.swift
//
//  ViewController.swift
//  DPIconFontExample-Swift
//
//  Created by 张鹏 on 2021/1/14.
//

import UIKit
import DPIconFont

class ViewController: UIViewController {
    
    /// 文本
    @IBOutlet weak var textLabel: UILabel!
    
    /// 图片
    @IBOutlet weak var imageView: UIImageView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.textLabel.font = ICFFont(66)
        self.textLabel.text = "\u{e602}"
        self.imageView.image = ICFSymbolImage("\u{e602}", 100)
    }
}

